src/bento/dashboardData.js

```diff
@@ -78,6 +78,27 @@ export const facetSectionVariables = {
     height: '5px',
     isExpanded: false,
   },
+  fileIds: {
```

Comment:
can we move it to separate variable saying search as we might not need if local search is disabled?

Reply:
I think we can, let me update.